config.js

```diff
@@ -0,0 +1 @@
+require('dotenv').config({ path: '.env' })
```

Comment:
Since the `.env` is in `.gitignore` (for good reasons), can we add a comment in the README on how a contributor can execute the tests locally?

Reply:
The nice thing is that a `.env` file is not a requirement of the test suite. Since we replace these with "fake-token" and are using nock to mock out our api requests, new contributors can clone the repo and run the test suite and it should be green "out of the box".

Having said that I want to add some documentation around adding a `.env` file and running the test suite with nock turned off. My plan is to address that early next week in a single PR once we've solidified how to best write these tests.

Reply:
Great!